tensorflow_serving/model_servers/main.cc

```diff
@@ -302,6 +303,7 @@ tensorflow::serving::PlatformConfigMap ParsePlatformConfigMap(
 int main(int argc, char** argv) {
   tensorflow::int32 port = 8500;
   bool enable_batching = false;
+  tensorflow::int32 per_process_gpu_memory_fraction = 100;
```

Comment:
Please have it default to 0, not 100. (0 is the default value in the tf config proto.)

Reply:
have fixed~